src/ContainerLogs.jsx

```diff
@@ -95,47 +95,40 @@ class ContainerLogs extends React.Component {
         }
         this.resize(this.props.width);
 
-        const logsData = {};
-        logsData.name = this.props.containerId;
-
-        varlink.connect(utils.getAddress(this.props.system), this.props.system)
-                .then(connection => {
-                    connection.monitor("io.podman.GetContainerLogs", logsData, this.onStreamMessage)
-                            .then(this.onStreamClose)
-                            .catch(e => {
-                                if (e.error === "ConnectionClosed")
-                                    this.onStreamClose();
-                                else
-                                    this.setState({
-                                        errorMessage: e.message,
-                                        streamer: null,
-                                    });
-                            });
-                    this.setState({
-                        streamer: connection,
-                        errorMessage: "",
-                    });
-                })
+        const connection = rest.connect(client.getAddress(this.props.system), this.props.system);
+        const options = {
+            method: "GET",
+            path: "/v1.12/libpod/containers/" + this.props.containerId + "/logs",
```

Comment:
Can container IDs contain spaces, slashes, question marks, colons, or any other magic characters? This could use encodeURIComponent() just to be on the safe side.

Reply:
no, it cannot, it is sha, so just `0-9a-f`.